src/reducers/profile-view.js

```diff
@@ -83,6 +85,36 @@ function profile(state: Profile | null = null, action: Action): Profile | null {
   }
 }
 
+/**
+ * This information is stored, rather than derived via selectors, since the coalesced
+ * function update would force it to be recomputed on every symbolication update
+ * pass. It is valid for the lifetime of the profile.
```

Comment:
I'm not sure I understand why that, but I'm sure you thought about it more than I did so I'll trust you... :)